src/container/cli.c

```diff
@@ -170,6 +174,40 @@ dup_with_default_ownership_props(daos_prop_t **prop_out, daos_prop_t *prop_in)
 		entries++;
 	}
 
+	if (!daos_prop_has_entry(prop_in, DAOS_PROP_CO_ROOTS)) {
+		uint64_t rf;
+
+		/**
+		 * If user did not provide his own root object, let's generate
```

Comment:
Question: Do we generally want to refrain from using gendered pronouns? Aside from political climate, I think it also makes it more formal.
E.g.
```suggestion
		 * If user did not provide their own root object, let's generate
```

Reply:
will fix too. Thanks.